docs/.custom_wordlist.txt

```diff
@@ -1,2 +1,23 @@
 OpenStack
-Upgrader
\ No newline at end of file
+Upgrader
+ussuri
```

Comment:
I think we should add these custom words when we actually include them in the documentation, especially since they are case sensitive and we don't know if we need to add more variations of them.

Reply:
Ok, closing it.